no attribute 'LazilyIndexedArray'

Looks like a new Xarray version has broken master.

```
=================================== FAILURES ===================================
_____________________________ test_xarray_dataset ______________________________
    @recorder.use_cassette('rap_compressed')
    def test_xarray_dataset():
        """Test getting data using xarray open_dataset over CDMRemote."""
        store = CDMRemoteStore('http://thredds-dev.unidata.ucar.edu/thredds/cdmremote/'
                               'grib/NCEP/RAP/CONUS_13km/RR_CONUS_13km_20150518_1200.grib2/GC')
>       ds = open_dataset(store)
siphon/cdmr/tests/test_xarray.py:20: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../virtualenv/python3.6.3/lib/python3.6/site-packages/xarray/backends/api.py:326: in open_dataset
    return maybe_decode_store(store)
../../../virtualenv/python3.6.3/lib/python3.6/site-packages/xarray/backends/api.py:238: in maybe_decode_store
    drop_variables=drop_variables)
../../../virtualenv/python3.6.3/lib/python3.6/site-packages/xarray/conventions.py:594: in decode_cf
    vars, attrs = obj.load()
../../../virtualenv/python3.6.3/lib/python3.6/site-packages/xarray/backends/common.py:217: in load
    for k, v in self.get_variables().items())
siphon/cdmr/xarray_support.py:54: in get_variables
    for k, v in self.ds.variables.items())
../../../virtualenv/python3.6.3/lib/python3.6/site-packages/xarray/core/utils.py:308: in FrozenOrderedDict
    return Frozen(OrderedDict(*args, **kwargs))
siphon/cdmr/xarray_support.py:54: in <genexpr>
    for k, v in self.ds.variables.items())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <siphon.cdmr.xarray_support.CDMRemoteStore object at 0x7f6b36eea6d8>
name = 'Geopotential_height_isobaric'
var = <siphon.cdmr.dataset.Variable object at 0x7f6b36ec0860>
    def open_store_variable(self, name, var):
        """Turn CDMRemote variable into something like a numpy.ndarray."""
>       data = indexing.LazilyIndexedArray(CDMArrayWrapper(name, self))
E       AttributeError: module 'xarray.core.indexing' has no attribute 'LazilyIndexedArray'
siphon/cdmr/xarray_support.py:48: AttributeError
==================== 1 failed, 289 passed in 137.13 seconds ====================

```

--- a/environment.yml
+++ b/environment.yml
@@ -29,6 +29,6 @@ dependencies:
   - flake8-quotes
   - pep8-naming
   - matplotlib
-  - xarray
+  - xarray >=0.10.2
   - cartopy
   - beautifulsoup4

--- a/setup.py
+++ b/setup.py
@@ -52,7 +52,7 @@ setup(
                  'flake8-copyright', 'flake8-docstrings', 'flake8-import-order',
                  'flake8-mutable', 'flake8-pep3101', 'flake8-print', 'flake8-quotes',
                  'pep8-naming',
-                 'vcrpy~=1.5,!=1.7.0,!=1.7.1,!=1.7.2,!=1.7.3', 'xarray>=0.6'],
+                 'vcrpy~=1.5,!=1.7.0,!=1.7.1,!=1.7.2,!=1.7.3', 'xarray>=0.10.2'],
         'doc': ['sphinx>=1.3,!=1.6.4', 'sphinx-gallery', 'doc8', 'recommonmark'],
         # SciPy needed for cartopy; we don't use cartopy[plotting] because
         # that will pull in GDAL.

--- a/siphon/cdmr/xarray_support.py
+++ b/siphon/cdmr/xarray_support.py
@@ -29,9 +29,15 @@ class CDMArrayWrapper(BackendArray):
 
     def __getitem__(self, item):
         """Wrap getitem around the data."""
-        item = indexing.unwrap_explicit_indexer(
-            item, self, allow=(indexing.BasicIndexer, indexing.OuterIndexer))
-        return self.get_array()[item]
+        item, np_inds = indexing.decompose_indexer(item, self.shape,
+                                                   indexing.IndexingSupport.BASIC)
+        with self.datastore:
+            array = self.get_array()[item.tuple]
+
+        if len(np_inds.tuple) > 0:
+            array = indexing.NumpyIndexingAdapter(array)[np_inds]
+
+        return array
 
 
 class CDMRemoteStore(AbstractDataStore):
@@ -45,7 +51,7 @@ class CDMRemoteStore(AbstractDataStore):
 
     def open_store_variable(self, name, var):
         """Turn CDMRemote variable into something like a numpy.ndarray."""
-        data = indexing.LazilyIndexedArray(CDMArrayWrapper(name, self))
+        data = indexing.LazilyOuterIndexedArray(CDMArrayWrapper(name, self))
         return Variable(var.dimensions, data, {a: getattr(var, a) for a in var.ncattrs()})
 
     def get_variables(self):

